src/app/pages/OnDemandRadioPage/index.jsx

```diff
@@ -195,7 +209,7 @@ OnDemandRadioPage.propTypes = {
     headline: string,
     summary: string,
     language: string,
-    episodeIsAvailable: bool,
+    episodeIsAvailable: string,
```

Comment:
Could use the `oneOf` validator here :)

https://reactjs.org/docs/typechecking-with-proptypes.html#proptypes

Reply:
ty, added 👍